irods/exception.py

```diff
@@ -67,8 +69,39 @@ def __init__(self, name, bases, attrs):
             iRODSExceptionMeta.codes[attrs['code']] = self
 
 
+class Errno:
+    def __init__(self,arg0,*args):
+        self.int_code = arg0
+    def __repr__(self):
+        e = self.int_code
+        try:
+            return self.__class__.__name__ + repr(tuple([e, errno.errorcode[e], os.strerror(e)]))
```

Comment:
I'm guessing this won't cause any issues on Windows?

Reply:
Did try out the errno's on Windows and they seem to correspond, at least for this one.  POSIX and all that.